core/ui/PageControls/export-twcore.tid

```diff
@@ -0,0 +1,19 @@
+title: $:/core/ui/Buttons/export-twcore
```

Comment:
Again, "twcore" should be spelt out in full as "tiddlywikicore" throughout this file